TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/Main.java

```diff
@@ -171,11 +174,46 @@ private static IConnectionCaptureFactory getConnectionCaptureFactory(Parameters
         }
     }
 
+    private static URI convertStringToUri(String uriString) {
```

Comment:
It's probably worth at least calling out in a comment how this is similar to the replayer.